app.py

```diff
@@ -22,8 +23,8 @@ def process_video(input_video_path):
     cfg.MODEL.WEIGHTS = "https://dl.fbaipublicfiles.com/densepose/densepose_rcnn_R_50_FPN_s1x/165712039/model_final_162be9.pkl"
     cfg.MODEL.DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
     predictor = DefaultPredictor(cfg)
-
-    # Open the input video
+    
+# Open the input video
```

Comment:
I believe these lines were edited accidentally.
```suggestion

    # Open the input video
```